### PR TITLE
Add Botble CMS version detection

### PR DESCRIPTION
**Pull request by sangnguyenplus**
Hi,

I want to add version to our CMS detection. Please check it.

download

Thank you so much!

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -5763,6 +5763,9 @@
       "cookies": {
         "botble_session": ""
       },
+      "headers": {
+        "CMS-Version": "^(.+)$\\;version:\\1"
```
Comment (jvoisin) [inline src/apps.json]: Please use `"CMS-Version": "^(.+)$\\;version:\\1;confidence:0"` to reduce the number of false-positives.

Comment (sangnguyenplus) [reply src/apps.json]: Hi,

I've just updated it.

Thanks for your feedback!

**Comment by sangnguyenplus** [comment]
@jvoisin Can you check it again?

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**